plugins/modules/aci_vrf_fallback_route.py

```diff
@@ -0,0 +1,391 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_vrf_fallback_route
+short_description: Manage VRF fallback route groups (fv:FBRGroup)
+description:
+- Manage VRF fallback route groups on Cisco ACI fabrics.
+- Fallback route groups are used to specify routes and next-hop addresses for VRFs.
+options:
+  tenant:
+    description:
+    - The name of the Tenant to which the VRF fallback route group belongs.
+    type: str
+    aliases: [ tenant_name ]
+  vrf:
+    description:
+    - The name of the VRF associated with the fallback route group.
+    type: str
+    aliases: [ context, vrf_name ]
+  vrf_fallback_route_group:
+    description:
+    - The name of the VRF fallback route group to configure.
+    type: str
+    aliases: [ name ]
+  fallback_route:
+    description:
+    - The fallback route (prefix address) for the VRF fallback route group.
+    - If not specified, the existing fallback route will remain unchanged.
+    type: str
+    aliases: [ prefix_address ]
+  fallback_members:
+    description:
+    - A list of fallback member IP addresses (next-hop addresses) for the VRF fallback route group.
+    - Members not in the list will be removed from the configuration.
+    type: list
+    elements: str
+    aliases: [ next_hop_address ]
+  description:
+    description:
+    - The description for the VRF fallback route group.
+    type: str
+    aliases: [ descr ]
+  state:
+    description:
+    - Use C(present) to create or update the fallback route group.
+    - Use C(absent) to delete the fallback route group.
+    - Use C(query) to retrieve information about the fallback route group.
```

Comment:
why is this changed? typically we have:

```suggestion
state:
    description:
    - Use C(present) or C(absent) for adding or removing.
    - Use C(query) for listing an object or multiple objects.
    type: str
    choices: [ absent, present, query ]
```